sass/layout/_markdown.scss

```diff
@@ -58,9 +58,8 @@ h6 {
         max-height: 500px;
     }
     div.markdown-inline-img {
-        vertical-align: middle;
-        display: inline-block;
-        width: inherit;
```

Comment:
This was causing https://mattermost.atlassian.net/browse/MM-16100. As this component wasn't is used in a lot of places there are places where there is no width to inherit causing a pop. My initial commit was to use `fill-content` property or `width: 100%` but it will cause pop with inline images as it will start with 100% width and then changes to inline image. 

The whole reason of adding this here was to fix the 10px pop caused by a margin property on markdown images https://github.com/mattermost/mattermost-webapp/pull/2885. which is now changed to added the margin on the svg placeholder inside .`markdown-inline-img`

Reply:
@migbot @enahum Can you guys give this another look?